sys/Makefile.dep

```diff
@@ -271,20 +271,26 @@ endif
 ifneq (,$(filter gnrc_sixlowpan_default,$(USEMODULE)))
   USEMODULE += gnrc_ipv6_nib_6ln
   USEMODULE += gnrc_sixlowpan
-  USEMODULE += gnrc_sixlowpan_frag
+  ifeq (,$(filter gnrc_sixlowpan_frag_sfr,$(USEMODULE)))
+    USEMODULE += gnrc_sixlowpan_frag
```

Comment:
That would be IMHO a separate PR. Either a dependent for this PR or as a follow-up, otherwise I believe the patch is too confusing. I would prefer a follow-up, as I don't know when I find time to work on that.